src/main/scala/nl/biopet/utils/tool/ToolCommand.scala

```diff
@@ -145,19 +145,34 @@ trait ToolCommand[Args] extends Logging {
   def example(args: String*): String = {
     cmdArrayToArgs(args.toArray)
 
-    exampleToMarkdown(args: _*)
+    exampleToMarkdown(false, args: _*)
```

Comment:
See comments below. I would prefer
```Scala
exampleToMarkdown(args: _*)
``` 
here

Reply:
also not possible ;)